deepcell_tracking/isbi_utils.py

```diff
@@ -40,32 +40,33 @@
 from deepcell_tracking.utils import load_trks
 
 
-def trk_to_isbi(track, path):
+def trk_to_isbi(track):
     """Convert a lineage track into an ISBI formatted text file.
 
     Args:
         track (dict): Cell lineage object.
-        path (str): Path to save the .txt file.
+
+    Returns:
+        isbi (arr): Array of ISBI dictionaries of each label.
```

Comment:
No need to name the return value in the docs, just the data type.

```suggestion
        array: Array of ISBI dictionaries of each label.
```